### PR TITLE
[YUNIKORN-804] ws/v1/clusters returns invalid JSON when there are mul…

### PR DESCRIPTION
**Pull request by chia7712**
…tiples partitions

### What is this PR for?
it returns "two arrays" when there are two partitions. We should merge them into single array. Also, the partition name should be included by each element.
This PR changes the API format of multiple partitions. However, it should be fine as we should not rely on invalid JSON.

### What type of PR is it?
* [x] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-804

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: pkg/webservice/dao/cluster_info.go [hunk 1]
```diff
@@ -18,6 +18,7 @@
 package dao
 
 type ClusterDAOInfo struct {
+	PartitionName         string `json:"partition"`
```
Comment (chia7712) [inline pkg/webservice/dao/cluster_info.go:21]: I will file another PR to update APIs example.

**Review by manirajv06 — CHANGES_REQUESTED** [review]
Can we create a new test method to keep the asserts rather than modifying TestGetPartitionQueuesHandler?

File: pkg/webservice/dao/cluster_info.go (hunk 1 above)
Comment (chia7712) [reply pkg/webservice/dao/cluster_info.go:21]: https://issues.apache.org/jira/browse/YUNIKORN-824

**Comment by chia7712** [comment]
> Can we create a new test method to keep the asserts rather than modifying TestGetPartitionQueuesHandler?

sure!

**Review by manirajv06 — APPROVED** [review]

**Pull request closed without merging**